### ISSUE TITLE
Floating Button is missing accessible name

### ISSUE THREAD

**Issue by jen-rasmussen** [pressidium/pressidium-cookie-consent#204]
**Describe the bug**
When the floating button is enabled, accessibility checkers indicate that the button is missing an accessible name. 

**Steps to reproduce**
1. Enable Plugin
2. Enable Floating Button
3. Run accessibility page check (such as Acquia Optimize)
4. See error (screenshot)

**Expected behavior**
The floating button should have an accessible name as to not trigger an accessibility error. 

**Screenshots**
Testing with Acquia Optimize shows: 
<img width="330" height="371" alt="Image" src="https://github.com/user-attachments/assets/0b7b68d2-1f3a-431c-9c11-2f27be89634b" />

**Environment (please complete the following information):**
 - Browser: Firefox
 - Extension: Acquia Optimize
 - Plugin version: 1.8.0
 - WordPress version: 6.9.4
 - PHP version: 8.4.21

**Server logs**
N/A

**Browser console logs**
N/A

**Checklist**
* [x] I have searched the [GitHub issues](https://github.com/pressidium/pressidium-cookie-consent/issues) to ensure this is not a duplicate
* [x] I have provided sufficient information to reproduce and understand the issue
* [x] I have tested the issue using the latest version of the plugin
* [x] I have included any relevant error messages, logs, or screenshots
* [x] I have reviewed the [wiki](https://github.com/pressidium/pressidium-cookie-consent/wiki) to ensure this issue is not related to a misconfiguration

**Additional context**
N/A

**Comment by over-engineer** [comment]
Closing as a duplicate of #163.

This issue has already been fixed since version `1.9.0`, so please **make sure you are testing with the latest version** of the plugin before opening a new issue.

Also, please take a moment to search through existing GitHub issues before creating a new one, as this helps avoid duplicate reports and makes it easier for us to track and prioritize unresolved problems.

These checks are included in the issue template to help keep issue tracking manageable and save time for both users and maintainers. In this case, it looks like those steps were skipped before opening the issue.

Thanks for understanding and appreciate the report nonetheless.

**Comment by jen-rasmussen** [comment]
My apologies for the duplicate, I must not have checked Closed issues. Thank you!

**Comment by over-engineer** [comment]
@jen-rasmussen No worries at all! If anything else comes up, feel free to open another issue 🙂
